src/commands/check.ts

```diff
@@ -14,10 +14,14 @@ export class CheckCommand extends Command {
     description: 'Name of the Leon instance.'
   })
 
+  static async run(leonInstance: LeonInstance): Promise<void> {
+    await leonInstance.check()
+  }
+
   async execute(): Promise<number> {
     try {
       const leonInstance = LeonInstance.get(this.name)
-      await leonInstance.check()
+      await CheckCommand.run(leonInstance)
```

Comment:
The run method is useful for the Run command but I find that using the static method here only create confusion. It's a bit less readable. I would prefer to let the check method of the leonInstance. It's only a personal opinion though, and is not blocking.

Reply:
Yep, I will change that. :+1: